addons/website_sale_stock/static/src/js/variant_mixin.js

```diff
@@ -61,6 +62,28 @@ VariantMixin._onChangeCombinationStock = function (ev, $parent, combination) {
         }
     }
 
+    // needed xml-side for formatting of remaining qty
+    combination.formatQuantity = (qty) => {
+        if (Number.isInteger(qty)) {
+            return qty;
+        } else {
+            const max_digits = 14;
```

Comment:
It's weird to define an arbitrary number here. I'd rather return `product.uom_id.rounding` in `combination_info`.